### PR TITLE
Added media_type to GET parameter

### PR DESCRIPTION
**Pull request by FanJiangCa**
DSP entities GET needs media_type in parameter.

### CODE REVIEW

**Comment by pepecommops** [comment]
Argo CI comment: Looks good to me! and your package is here <a href=https://nexus.pepstaging.com/#browse/browse:pypi-dev:amazon-advertising-api-python/0.0.0-devd2a5e8d>0.0.0-devd2a5e8d</a>

**Comment by pepecommops** [comment]
Argo CI comment: Looks good to me! and your package is here <a href=https://nexus.pepstaging.com/#browse/browse:pypi-dev:amazon-advertising-api-python/0.0.0-dev2685acc>0.0.0-dev2685acc</a>

File: amazon_advertising_api/advertising_api.py [hunk 1]
```diff
@@ -1094,6 +1094,23 @@ def get_report(self, report_id, report_type=None):
                 res = self._download(location=body['location'])
         return res
 
+    def get_dsp_entity(self, interface, media_type):
```
Comment (rahulnbhandari) [inline amazon_advertising_api/advertising_api.py:1097]: can may be add unit test for this?

Comment (rahulnbhandari) [reply amazon_advertising_api/advertising_api.py:1097]: i suppose this dont have unit test as part of c

**Review by rahulnbhandari — APPROVED** [review]

**Pull request merged**